### DISCUSSION TITLE
Equal Constraining All Selected Entities To Last Selected Entity

### DISCUSSION THREAD

**Discussion by TheMakerTales** [hlorus/CAD_Sketcher#209] [Feature Requests]
A little hard to write more in detail here but as the title suggests it's the ability to select lots of lines and then when using an equal constraint. This would make all selected lines equal to the last selected line.

**Comment by calebmeyer** [comment]
Yeah, my first project was making Tangrams to 3d print for my kid, and having like 6 equals constraints on one side of the square got pretty messy.

**Reply by TheMakerTales** [reply]
It's tricky really as you want to be able to see how many lines are driven by the constraint yet at the same time not clutter up the view. I have a feeling that this might be solved by Blender drivers in the future when we can set variables in a custom property tho. However in the meantime maybe once there is 3+ of the same constraint in one location it turns into some tight of dropdown selection and it's just the single constraint will a little number next to it?

**Comment by optimiseddesign** [comment]
I like it, though would intuitively think they should be made the same as the _first_ one I selected. After the first, they're the clones!
